pennylane/io/io.py

```diff
@@ -842,8 +842,8 @@ def from_qasm3(quantum_circuit: str, wire_map: dict = None):
     Converts an OpenQASM 3.0 circuit into a quantum function that can be used within a QNode.
 
     .. note::
-        The following OpenQASM 3.0 gates are not supported: sdg, tdg, cu. The remaining standard library gates are
-        all supported. End statements, robust variables, subroutines, control flow, measurements, built-in mathematical
+        The following OpenQASM 3.0 gates are not supported: sdg, tdg, cu. The remaining standard library gates as well
+        as end statements are supported. Robust variables, subroutines, control flow, measurements, built-in mathematical
```

Comment:
```suggestion
        as ``end`` statements are supported. Robust variables, subroutines, control flow, measurements, built-in mathematical
```

I _think_ this gets backticks?

Reply:
I guess `end;` *is* a statement in QASM 3.0 ...